index.html

```diff
@@ -0,0 +1,165 @@
+            <img class="sponsor-logo" src="images/Gazprom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/freedom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/Besthomes.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/loremipsum.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/wildlife.jpg" alt="sponsor-logo">
+        </div>
+    </div>
+
+
+    <section id="about2">
+        <div class="section1">
+            <div class="header1-div"><h2>"Trees for better lives"</h2></div>
+            <div class="theme-header"><h2>Climate Change Global Conference-2022</h2></div>
+            <div class="about2-quote"><p>The network is organizing a continental conference in Nairobi Kenya for the 12 member countries. The conference topics are on climate change, ToP methods in Africa, Courage to Lead, institutional Sustainability and Fundraising. The conference is scheduled to be held from 22nd to 29th October 2022.</p></div>
+            <div class="contact-details">please contact us per Email for any further questions <br>about cc global conference-2022 <br><strong>collinskiprtich@gmail.com</strong></div>
+        </div>
+        <div class="section2">
+            <div><h3>Climate Change Global summit 2022 Logo </h3></div>
+            <div><p class="no-border">The logo for the Climate Change global summit was decided through the logo competition from 1.August- 3.November </p> </div>
+            <div class="logo-border"><img src="images/my-icon.png" alt=""></div>
+    </div>
+
+    <div class="section3">
+        <div><h2>See The Past CC Global conferences</h2></div>
+        <div><p>Take a look at som of the pictures from previous events held in Bueno Aires, Argentina</p></div>
+       <div class="container-previous-images">
+        <div class="previous-events-images-container">
+            <img class="previous-image" src="images/un2.jpg" alt="">
+            <div class="centered">2019 <br> Climate conferece Summit <br> Bueno Aires, Argentina</div>
+        </div>
+        <div class="previous-events-images-container">
+            <img class="previous-image" src="images/united-nations.jpg" alt="">
+            <div class="centered">2019 <br> Climate conferece Summit <br> Bueno Aires, Argentina</div>
+        </div>
+    </div>
+       </div> 
+
+    <div class="section4">
+        <div><h2>Partners</h2></div>
+        <div class="sponsor-container ">
+            <img class="sponsor-logo" src="images/Gazprom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/freedom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/Besthomes.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/loremipsum.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/wildlife.jpg" alt="sponsor-logo">
+        </div>
+    </div>
+    </section>
```

Comment:
- [ ]  As stated the project requirement The pages should look almost identical to the original design , So Kindly remove the "About " page sections at "Home" Page Just Like the Design. After the "Partner" section there must be "Footer" section Just Like design.

Your Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/203768341-94993d75-b6f2-4391-bb2a-19097b098a78.PNG)

The Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/203768751-fd6f4523-0a4b-43dd-b4bf-c2c8f47a1087.PNG)